back/test/endpoint_tests/test_product.py

```diff
@@ -30,6 +31,7 @@ def test_product_query(read_only_client, default_product, default_size, another_
         "price": default_product["price"],
         "gender": "Women",
         "createdBy": {"id": str(default_product["created_by"])},
+        "deleted": None,
```

Comment:
Should it be worth adding a test case for a soft-deleted product somewhere?

Reply:
Sure, there's already a deleted product in the test data (`back/test/data/product.py`). In the first test in the current file you can add another request to query the deleted product and it's `deletedOn` field. :muscle: